tmt/steps/provision/mrack.py

```diff
@@ -376,6 +377,24 @@ def _transform_virtualization_is_virtualized(
     return _transform_unsupported(constraint, logger)
 
 
+def _transform_virtualization_hypervisor(
+        constraint: tmt.hardware.FlagConstraint,
```

Comment:
@happz, isn't this rather `TextConstraint`?

Reply:
Oh yeah, very much so.

Reply:
Ok, will adjust, so that we can merge this.